string frame_id
string id 
---
std_msgs/Empty empty